src/plugins/advanced_settings/server/plugin.ts

```diff
@@ -21,6 +22,8 @@ export class AdvancedSettingsServerPlugin implements Plugin<object, object> {
 
     core.capabilities.registerProvider(capabilitiesProvider);
 
+    core.uiSettings.setAllowlist(ALL_ALLOWLIST_SETTINGS);
```

Comment:
This is added here only for testing. It will be called from the serverless plugin instead once we register the Advanced settings app there.